### PR TITLE
Report Z changes

### PR DESCRIPTION
**Pull request by BarinderBanwait**
Code changes due to Report Z. I think this is complete. It is mostly just annotations. @koffie you should actually review this.

Let's merge this to main when you're happy. We can then create a `v2` branch corresponding to v2 of the paper. The v2 branch should then be considered frozen, and any subsequent development be done on main.

### CODE REVIEW

File: computations/x1_13.m
Comment (koffie) [inline computations/x1_13.m:9]: maybe leave a comment about krumms thesis why we can restrict to d > 0 and d = 1 mod 8.

File: magma_scripts/MWSieve-x1_13.m
Comment (koffie) [inline magma_scripts/MWSieve-x1_13.m:38]: We are claiming results unconditional on GRH in our paper. So we should either justify why the computed results are independent of GRH, or remove this statement. So I think how to fix this, is to move the rank computations here (before setting the class group bounds). And then make a remark that once we computed the ranks, then it is ok to use class group bounds, since at that point we only use it to speed up the search of generators.

File: computations/x1_13.m
Comment (koffie) [inline computations/x1_13.m]: I think this entire code block for rank computations can go. We don't need it here, we are doing rank computations anyway in the MW sieve file. Unless we have a specific thing in our paper referring to this.

File: magma_scripts/MWSieve-x1_13.m
Comment (koffie) [inline magma_scripts/MWSieve-x1_13.m]: We don't seem to have a log file capturing the output of this run. Also similar to the x1_13.m and other files I really much would like to keep todo and done lists that are printed at the end so that it is easy to see what the outcome of the computation was.

File: computations/x1_16_quadratic_chabauty.m
Comment (koffie) [inline computations/x1_16_quadratic_chabauty.m:1]: Actually I'm for removing it. Without any computation, just pure thought it is already clear that the endomorphism ring is going to be Z[i] and that hence quadratic chabauty on the curve itself is not going to work. As mentioned in conversations before, my idea was never to do quadratic chabauty on the the twists of X1(16) themselves, but to first do a two descent to make us end up on a curve where quadratic chabauty is possible.

File: computations/x1_16_chabauty.m
Comment (koffie) [inline computations/x1_16_chabauty.m]:
```suggestion
// https://magma.maths.usyd.edu.au/magma/handbook/text/1566#18074
```

File: computations/x1_16_chabauty.m
Comment (koffie) [inline computations/x1_16_chabauty.m]: The wording could be clearer, first of all it is about the "fake two selmer set" not the two selmer set. Secondly this function doesn't help us represent the elements of the two selmer set. It is actually the function that is returned by this function that does this. I propose something like:

Magma's TwoCoverDescent function returns a function AtoHk that maps elements from some algebra A to the fake 2-Selmer set Hk. This function returns the inverse of this map. Elements in the algebra A can be used to construct the two-cover of the corresponding element in the fake 2-Selmer set.

File: computations/x1_16_chabauty.m
Comment (koffie) [inline computations/x1_16_chabauty.m]: This could theoretically be a pointless genus 1 curve if the degree of g is even.

File: computations/x1_16_chabauty.m
Comment (koffie) [inline computations/x1_16_chabauty.m]:
```suggestion
    // If we can't find a point, we return
    // technically we could try to proceed and show that E has no points,
    // but since E is covered by an everywhere locally solvable curve
    // this will not be easy.
    
```

File: computations/x1_16_chabauty.m
Comment (koffie) [inline computations/x1_16_chabauty.m]: We should remove this failed cases list. This was only something that was nice to have here while tweaking the code.

File: computations/x1_16_chabauty.m
Comment (koffie) [inline computations/x1_16_chabauty.m]: genus 1 curves...

File: magma_scripts/MWSieve-x1_18.m
Comment (koffie) [inline magma_scripts/MWSieve-x1_18.m:38]: Same comment as for the X_1(13) mw sieve.

File: magma_scripts/MWSieve-x1_18.m
Comment (koffie) [inline magma_scripts/MWSieve-x1_18.m:137]: also print a list of succesfull and failed d's at the end, so that it is easy to see what the result was. Also we should add a log file containing the output of running this.

**Review by koffie — CHANGES_REQUESTED** [review]

**Comment by koffie** [comment]
I didn't manage to review everyhing yet. But at least I left my comments on all magma files except the ABC triples file. The rest will come after the weekend.

**Comment by koffie** [comment]
p.s. I agree with having version that should not be modified anymore, however we shouldn't do it with branches. Instead once we merged everything into main we should just create a tag. Branches are  not really intended for immutable stuff, tags are better suited for this.

File: computations/x1_13.m
Comment (koffie) [reply computations/x1_13.m:9]: done

File: computations/x1_13.m
Comment (koffie) [reply computations/x1_13.m]: done

File: computations/x1_16_chabauty.m
Comment (koffie) [reply computations/x1_16_chabauty.m]: done

File: computations/x1_16_chabauty.m
Comment (koffie) [reply computations/x1_16_chabauty.m]: done

File: computations/x1_16_chabauty.m
Comment (koffie) [reply computations/x1_16_chabauty.m]: done

File: computations/x1_16_chabauty.m
Comment (koffie) [reply computations/x1_16_chabauty.m]: done

File: computations/x1_16_chabauty.m
Comment (koffie) [reply computations/x1_16_chabauty.m]: done

File: computations/x1_16_chabauty.m
Comment (koffie) [reply computations/x1_16_chabauty.m]: done

File: magma_scripts/MWSieve-x1_13.m
Comment (koffie) [reply magma_scripts/MWSieve-x1_13.m:38]: done

File: computations/x1_16_quadratic_chabauty.m
Comment (koffie) [reply computations/x1_16_quadratic_chabauty.m:1]: done

File: magma_scripts/MWSieve-x1_18.m
Comment (koffie) [reply magma_scripts/MWSieve-x1_18.m:38]: done

File: magma_scripts/MWSieve-x1_18.m
Comment (koffie) [reply magma_scripts/MWSieve-x1_18.m:137]: the code changes are done. however adding the log file is a separate issue now: https://github.com/isogeny-primes/quadratic-torsion/issues/5 since this computation is still running

File: magma_scripts/MWSieve-x1_13.m
Comment (koffie) [reply magma_scripts/MWSieve-x1_13.m]: code changes are done. adding the log is #5

**Comment by BarinderBanwait** [comment]
@koffie ended up addressing his own review comments since I was away last week. This PR can be merged once #4 is merged.

**Pull request merged**
